interpreter/program.go

```diff
@@ -36,8 +36,12 @@ type Program interface {
 	// container is used to resolve type names and identifiers.
 	Container() string
 
-	// GetInstruction returns the instruction at the given expression id.
-	GetInstruction(exprId int64) Instruction
+	// Return the runtime expression id corresponding to the expression id from
+	// the AST.
+	GetRuntimeExpressionId(exprId int64) int64
```

Comment:
Prefer organizing exported names alphabetically. The code doesn't do this consistently, but it's something I've been working on getting better about doing in recent changes.